components/gitpod-protocol/src/experiments/configcat-server.ts

```diff
@@ -35,6 +35,7 @@ export function getExperimentsClientForBackend(): Client {
         requestTimeoutMs: 2000,
         logger: configcat.createConsoleLogger(LogLevel.Error),
         maxInitWaitTimeSeconds: 0,
+        baseUrl: process.env.CONFIGCAT_BASE_URL,
```

Comment:
Do we need to check this value is set, and if not, use the `alwaysReturningDefault`?